packages/venia-concept/prettier.config.js

```diff
@@ -0,0 +1,5 @@
+const config = {
+    singleQuote: true
+};
+
+module.exports = config;
```

Comment:
Should this file be located in the `pwa-studio` root?

Reply:
Just to put context in this convo for later -- my intent was that `venia-concept`, when used as the root "theme" from the scaffolding script, would provide this config. I wasn't sure if the scaffolding script already did that work and copied the one in the root folder. If that is the case, then I can delete this file.

Reply:
This is the right move. Keeping these two files in sync shouldn't be a big problem, because they should VERY rarely change.